artifactory/commands/buildinfo/publish.go

```diff
@@ -91,12 +94,51 @@ func (bpc *BuildPublishCommand) Run() error {
 	if err != nil {
 		return err
 	}
+
+	buildTime, err := time.Parse(buildinfo.TimeFormat, buildInfo.Started)
+	if errorutils.CheckError(err) != nil {
+		return err
+	}
+
+	artVersion, err := servicesManager.GetVersion()
+	if err != nil {
+		return err
+	}
+	artVersionToStr, err := strconv.Atoi(artVersion[0:1])
+	if err != nil {
+		return err
+	}
+	log.Info(bpc.printBuildInfoLink(artVersionToStr, buildTime))
```

Comment:
Let's move the following code into a private ```logBuildInfoUiUrl``` function. Shorter functions are preferred when possible and makes sense.consist
Also please mind the new-lines. Our code should look consistent -  why then is a new line before - 
```artVersion, err := servicesManager.GetVersion()```
and not before 
```artVersionToStr, err := strconv.Atoi(artVersion[0:1])```
?